server/api/event/index.js

```diff
@@ -56,39 +56,34 @@ router.get('/user/:userId', function(req, res) {
 // Create a new event and return event
 router.post('/create', function(req, res) {
   var currentDate = new Date();
-  if (req.body.userId) {
-    // Make sure user exists
-    db.User.findOne({
-      where: {
-        id: req.body.userId
-      }
-    }).then(function (user) {
-      if (user) {
-        // User exists, continue to create event
-        db.Event.create({
-          name: req.body.name || '',
-          // If no start time specified, default to now
-          start: req.body.start || currentDate,
-          // If no end time specified, default to 24 hours from now
-          end: req.body.end || new Date(currentDate.getTime() + 60 * 60 * 24 * 1000),
-          location: req.body.location || '',
-          action: req.body.action || '',
-          minParticipants: req.body.minParticipants || null,
-          maxParticipants: req.body.maxParticipants || null
-        }).then(function(event) {
-          // After creating the event, associate the event with the user as a shepherd
-          user.addShepherdEvent(event).then(function(shepherdEvent) {
-            event.addShepherd(user).then(function(){
-              res.json(event);
-            });
+  
```

Comment:
Would the user already exist in the database when he/she tries to create a new event though?